packages/puppeteer/src/getConfiguration.ts

```diff
@@ -30,6 +30,28 @@ export const getConfiguration = (): Configuration => {
     configuration.defaultProduct ??
     'chrome') as Product;
 
+  // Support the CHROME_BIN/FIREFOX_BIN convention found elsewhere. All of
+  // these, are lowest precedent and maintain current behaviors
```

Comment:
it looks like some tests fail with this (not sure why though)

Reply:
Failure reason https://github.com/puppeteer/puppeteer/issues/9418#issuecomment-1360235835